frontend/src/bundles/common/components/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import clsx from 'clsx';
+import { Tooltip as ReactTooltip } from 'react-tooltip';
+
+import QuestionIcon from '~/assets/img/question-circle.svg';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    className?: string,
+    text: string
+};
+
+const Tooltip: React.FC<Properties> = ({
+    className = '',
+    text
+}) => {
+    return <div className={clsx(className, styles.tooltip__container)}>
+        <img src={QuestionIcon} alt='tooltip' className={styles.tooltip__image} />
```

Comment:
It's redundant. Tooltip may be used anywhere

Reply:
Can you please clarify

Reply:
I think Vitaliy means that the tooltip can wrap any element; we can use it not only for the QuestionIcon but for any component. So, you need to remove <img /> from the tooltip component and pass the component that will be wrapped by the tooltip through props.